src/runtime_src/core/pcie/driver/linux/xocl/xocl_drv.h

```diff
@@ -943,6 +943,9 @@ struct xocl_ps_funcs {
 #define	xocl_ps_check_healthy(xdev)			\
 	(PS_CB(xdev, check_healthy) ? PS_OPS(xdev)->check_healthy(PS_DEV(xdev)) : true)
 
+#define xocl_ps_sched_on(xdev)	\
+	(XOCL_DSA_IS_VERSAL(xdev) || XOCL_DSA_IS_MPSOC(xdev))
```

Comment:
Please use "!XOCL_DSA_MB_SCHE_OFF(xdev) && (XOCL_DSA_IS_VERSAL(xdev) || XOCL_DSA_IS_MPSOC(xdev)
).
As Brain explained, MB_SCHE_OFF should be the top level switch for development or debug purpose.